caf-audit-webserviceclient/src/main/java/com/hpe/caf/auditing/webserviceclient/WebServiceClientAuditEventBuilder.java

```diff
@@ -18,7 +18,7 @@
 import com.hpe.caf.auditing.AuditCoreMetadataProvider;
 import com.hpe.caf.auditing.AuditEventBuilder;
 import com.hpe.caf.auditing.AuditIndexingHint;
-import com.hpe.caf.auditing.elastic.ElasticAuditConstants;
+import com.hpe.caf.auditing.constants.CafAutditConstants;
```

Comment:
Ah... I see this is where the `ElasticAuditConstants` confusion has come in - a pre-exising case where somebody has referenced ES Constants from the WS implementation.  This was always wrong.  The WS implementation need to adhere to the WS contract - not to the internal ES representation.  Just take a copy of the relevant constants for now.